library/src/main/java/com/github/amlcurran/showcaseview/ShowcaseView.java

```diff
@@ -343,6 +340,24 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
     }
 
     private static void insertShowcaseView(ShowcaseView showcaseView, Activity activity) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            Display display = ((WindowManager) activity.getSystemService(Context.WINDOW_SERVICE)).getDefaultDisplay();
+            int myOrientation = display.getRotation();
+            switch (myOrientation) {
+                case Surface.ROTATION_0:
+                    showcaseView.setPadding(0, 0, 0, ApiUtils.getSoftButtonsBarSizePort(activity));
+                    break;
+                case Surface.ROTATION_90:
+                    showcaseView.setPadding(0, 0, ApiUtils.getSoftButtonsBarSizeLand(activity), 0);
+                    break;
+                case Surface.ROTATION_180:
+                    showcaseView.setPadding(0, 0, ApiUtils.getSoftButtonsBarSizeLand(activity), 0);
```

Comment:
Ok, thanks, so it seems that my approach is still not enough generic.

We can try smth like this:
`if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
    showcaseView.setPadding(0, 0, ApiUtils.getSoftButtonsBarSizeLand(activity), ApiUtils.getSoftButtonsBarSizePort(activity));
}`